src/index.js

```diff
@@ -19,16 +19,5 @@
  */
 import ReactDOM from "react-dom";
 import App from "./App";
-import Map from "./Map";
 
-ReactDOM.render(
-  <div>
-    <div style={{ width: "100%" }}>
-      <App />
-    </div>
-    <div style={{ marginLeft: "300px" }}>
```

Comment:
At one point I decided to put <Map> as _not_ a child of <App> due to performance issues (I think related to the dom up app changing and the map component getting updated too frequently). 

This was likely actually in fleet-debuger ... and there's probably a better way to work around those problems.    The cleanup you've done here is awesome.